index.adoc

```diff
@@ -1,8 +1,8 @@
 
 [#idp-mgmt-docs]
-= Product Documentation for identity configuration management for Kubernetes Technology Preview
+= The identity configuration management for Kubernetes documentation (Technology Preview)
 
-TODO
+The identity configuration management for Kubernetes is a set of software operators that enhance cluster fleet management. The identity configuration management for Kubernetes supports Red Hat OpenShift Container Platform and identity configuration management across clouds and data centers.
```

Comment:
I'm not so sure about putting "The" in front of our operator name. I think we could just capitalize "Identity" in this case as the first word of the sentence. If you're ok with it I can adjust it in a future pass.